extensions/amp-story/1.0/amp-story.js

```diff
@@ -1449,9 +1450,8 @@ export class AmpStory extends AMP.BaseElement {
     this.storeService_.dispatch(Action.TOGGLE_UI, uiState);
 
     if (uiState !== UIType.MOBILE || this.isLandscapeSupported_()) {
-      // TODO: Rename the TOGGLE_LANDSCAPE action. (#19670)
```

Comment:
Remember to close #19670 when merging this

Reply:
Oh thanks, done.